src/visualization/types/SimpleTable/PagedTable.tsx

```diff
@@ -43,7 +43,7 @@ const measurePage = (
   let signature
 
   while (rowIdx < result.table.length) {
-    if (result.table.columns.table.data[rowIdx] !== currentTable) {
+    if (result.table.columns?.table?.data?.[rowIdx] !== currentTable) {
```

Comment:
`currentTable` starts as `undefined` which we do not change. The left side can now be `undefined` due to optional chaining. This will cause the entire `if` block to be skipped, which is fine because `result.table` has no `columns to work with.